codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -246,7 +235,7 @@ public void addFlowControlled(Http2Stream stream, FlowControlled frame) {
      * For testing purposes only. Exposes the number of streamable bytes for the tree rooted at
      * the given stream.
      */
-    int streamableBytesForTree(Http2Stream stream) {
+    long streamableBytesForTree(Http2Stream stream) {
```

Comment:
Realized when I was adding the `totalPendingBytes` that this is subject to overflow as an `int`.